app/src/main/java/com/epam/themes/androidcomponents/SimpleService.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.themes.androidcomponents;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.widget.Toast;
+
+public class SimpleService extends Service {
+
+    private SimpleBinder mSimpleBinder;
+    private Object mResult;
+
+    @Override
+    public int onStartCommand(final Intent intent, final int flags, final int startId) {
+        Toast.makeText(this, "Service was started", Toast.LENGTH_LONG).show();
+
+        mResult = new Object();
+
+        sendBroadcast(new Intent("com.epam.CUSTOM_ACTION") {
```

Comment:
such things usually moved to one class/interface and should be reused. otherwise changes in one place will brake logic